File: Programming Fundamentals/Labs And Homeworks/DataTypesAndVariablesExercise/SumDigits.java
package dataTypesAndVariablesExercise;

import java.util.Scanner;

public class SumDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = Integer.parseInt(scanner.nextLine());
        int lastDigit = number % 10;
        int sum = 0;

        while (number > 0) {
            sum += lastDigit;
            number = number / 10;
            lastDigit = number % 10;
        }
        System.out.println(sum);
    }
}

File: JS Front-End/JS Syntax Fundamentals - Exercise/sumDigits.js
function sumDigits(digit) {
    let sum = 0;
    let arr = Array.from(String(digit));
    for (let i = 0; i < arr.length; i++) {
        sum += Number(arr[i]);
    }

    console.log(sum);
}

sumDigits(245678);